src/components/Meme.js

```diff
@@ -0,0 +1,78 @@
+import React, { useEffect, useState } from 'react';
+
+const Meme = () => {
+  const [meme, setMeme] = useState({ topText: '', buttomText: '', randomImage: '' });
+  const [allMemeImage, setAllMemeImage] = useState([]);
+
+  useEffect(() => {
+    const getMeme = async () => {
+      const res = await fetch('https://api.imgflip.com/get_memes');
+      const data = await res.json();
+      setAllMemeImage(data.data.memes);
```

Comment:
I think using the extraction could look much cleaner and less redundant by improving your code to look like this: 
``` suggestion
      const { data } = await res.json();
      setAllMemeImage(data.memes);
```